disttask/loaddata/subtask_executor.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loaddata
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/disttask/framework/proto"
+	"github.com/pingcap/tidb/disttask/framework/scheduler"
+	"github.com/pingcap/tidb/executor/importer"
+	"github.com/pingcap/tidb/util/logutil"
+	"go.uber.org/zap"
+)
+
+// ImportSubtaskExecutor is a subtask executor for load data.
+type ImportSubtaskExecutor struct {
+	task *MinimalTaskMeta
+}
+
+// Run implements the SubtaskExecutor.Run interface.
+func (e *ImportSubtaskExecutor) Run(ctx context.Context) error {
+	logutil.BgLogger().Info("subtask executor run", zap.Any("task", e.task))
```

Comment:
Hope framework will provide logger so we can filter a label to find all related log